### PR TITLE
Rename and reorganize some pieces

### PR DESCRIPTION
**Pull request by pnwpedro**
This PR is a no-op. It reorganizes some code and renames some files and vars, but doesn't handle _all_ FQL X renames. That will come later.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,13 +36,13 @@ $ yarn add @fauna-labs/serverless-fauna
 
 > NOTE: This package has not reached a 1.0 release yet. Minor version releases may still contain breaking changes. If you wish, you can restrict your projects to only accepting patch updates by prefacing the version number with a `"~"` in your `package.json` file. For example, `"~0.2.0"` or `"~0.1.6"`
 
-## FQL X (beta)
+## FQL 10 (beta)
```
Comment (macmv) [inline README.md:39]: we need a consistent naming scheme here. in the database we have this ticket, to fix the naming: https://faunadb.atlassian.net/browse/ENG-4996, which is unclear, so I'm not sure what the right name is (FQL v10 vs FQL 10). I think it makes sense to double check the names with product, and its a good excuse to get a consistent set of naming rules in place.

**Review by macmv — COMMENTED** [review]
rename looks good, but I think the actual way we refer to FQL 10 needs to be validated by product. I suspect this will only change the README, I'd just like to avoid renaming everything a second time.

File: index.js [hunk 2]
```diff
@@ -71,11 +73,12 @@ class ServerlessFaunaPlugin {
       removeCommands.push(remove);
     }
 
-    const faunaCommands = new FaunaCommands(
-      this.config,
+    const faunaCommands = new FaunaCommands({
+      config: this.config,
```
Comment (pnwpedro) [inline index.js:77]:
```suggestion
    const faunaCommands = new FaunaCommands(
      this.config,
```

File: index.js [hunk 3]
```diff
@@ -71,11 +73,12 @@ class ServerlessFaunaPlugin {
...
       deployCommands,
-      removeCommands.reverse()
-    );
+      removeCommands: removeCommands.reverse(),
+      options: this.options,
+    });
```
Comment (pnwpedro) [inline index.js:81]:
```suggestion
      removeCommands.reverse()
    );
```

File: README.md (hunk 1 above)
Comment (pnwpedro) [reply README.md:39]: Actually, I'll go ahead and revert the readme for now

**Review by macmv — APPROVED** [review]
shipit, doing the README fixes later seems reasonable

**Pull request merged**